src/Listener.php

```diff
@@ -148,17 +150,27 @@ public function process()
                     APILogger::addInfo('HoldRequestResult', $holdRequestResult);
 
                     if ($holdRequestResult->isSuccess() === false) {
-                        // TODO: Fetch e-mail from patron record
-                        APILogger::addInfo('E-mail time', $holdRequestResult->getHoldRequest()->getDocDeliveryData()->getEmailAddress());
+                        $holdRequest = HoldRequestClient::getHoldRequestById($holdRequestResult->getHoldRequestId());
+                        APILogger::addInfo('HoldRequest', $holdRequest);
+
+                        $patron = PatronClient::getPatronById($holdRequest->getPatron());
+
+                        if($holdRequest->getRecordType() === 'i')
+                        {
```

Comment:
I think PSR-2 dicates a `if` should look like:

```
<?php
if ($expr1) {
    // if body
} elseif ($expr2) {
    // elseif body
} else {
    // else body;
}
```

See: http://www.php-fig.org/psr/psr-2/